docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. See [Section 7.1.4.1](#7141-name-encoding) |
+| \ \ \ \ `validity`             | __**TBD**__ |
+| \ \ \ \ `subject`              | See [Section 7.1.2.2.1](#71221-cross-certified-subordinate-ca-naming) |
+| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.2.2](#71222-cross-certified-subordinate-ca-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.2.1 Cross-Certified Subordinate CA Naming
+
+Provided that the Issuing CA has confirmed that the existing CA Certificate was issued in compliance with the then-current version of the Baseline Requirements, the Issuing CA MAY deviate from the requirements in [Section 7.1.4](#714-name-forms) as follows:
+
+The encoded `subject` name shall be byte-for-byte identical to the encoded `subject` name of the existing CA Certificate.
+
+##### 7.1.2.2.2 Cross-Certified Subordinate CA Extensions
+
+The acceptable extensions and the requirements for those extensions in a Cross-Certified Subordinate CA vary based on whether or not the Subordinate CA is issued to and operated by the same organization as the Issuing CA or an Affiliate of the Issuing CA organization.
+
+Table: Extensions when the Subordinate CA is operated by the Issuing CA or an Affiliate of the Issuing CA.
+
+| __Extension__                     | __Presence__    | __Critical__          | __Description__ |
+| ----                              | -               | -                     | ----- |
+| `authorityKeyIdentifier`          | MUST            | N                     | See [Section 7.1.2.9.2](#71292-authority-key-identifier) |
+| `basicConstraints`                | MUST            | Y                     | See [Section 7.1.2.8.3](#71283-basic-constraints) |
+| `certificatePolicies`             | MUST            | N                     | See [Section 7.1.2.8.4](#71284-certificate-policies---affiliated-ca) (Affiliated CA) |
+| `crlDistributionPoints`           | MUST            | N                     | See [Section 7.1.2.9.3](#71293-crl-distribution-points) |
+| `keyUsage`                        | MUST            | Y                     | See [Section 7.1.2.8.6](#71286-key-usage) |
+| `subjectKeyIdentifier`            | MUST            | N                     | See [Section 7.1.2.9.5](#71295-subject-key-identifier) |
+| `extKeyUsage`                     | SHOULD[^eku_ca] | -                     | See [Section 7.1.2.2.3](#71223-extended-key-usage---unrestricted-affiliated-cross-certified-ca) |
```

Comment:
Resolving this as WONTFIX/WorkingAsIntended - this is intentional here, and does not have any normative change upon CAs, since both MAY and SHOULD unfortunately permit ignoring this. However, in line with browser expectations, this is very much a SHOULD to appropriately constrain CAs, as noted in the footnote.